ow_regolith/src/RegolithSpawner.cpp

```diff
@@ -230,8 +231,9 @@ bool RegolithSpawner::spawnRegolithInScoop(bool with_pushback)
 
   wrench_msg.request.body_name         = body_name.str();
   
-  // negative duration means the force is applied until we clear it
-  wrench_msg.request.duration          = ros::Duration(-1);
```

Comment:
I believe this is allowed, though I am unable to find documentation about it.

In any case, I got the same crash after starting the first grind maneuver in ReferenceMission1.plx.

Reply:
Negative duration is at least supported by [gazebo_msgs/ApplyBodyWrench](http://docs.ros.org/en/noetic/api/gazebo_msgs/html/srv/ApplyBodyWrench.html) as a way to indefinitely apply a wrench, but it's always possible the writers of this services assumed Duration was capable of something its not.